backend/package.json

```diff
@@ -106,7 +106,7 @@
 		"strip-ansi": "^6.0.0",
 		"titleize": "^1.0.1",
 		"ws": "^7.2.5",
-		"yeoman-environment": "2.10.0"
+		"yeoman-environment": "2.8.0"
```

Comment:
Was there a regression in 2.10.0? Why downgrade?

Reply:
I will not downgrade